lib/redis.js

```diff
@@ -163,7 +164,8 @@ Redis.defaultOptions = {
   enableReadyCheck: true,
   autoResubscribe: true,
   autoResendUnfulfilledCommands: true,
-  lazyConnect: false
+  lazyConnect: false,
+  readOnly: false
```

Comment:
Since `readOnly` only works in cluster mode, we'd better only have it in cluster.js and listen to the `ready` event in the `createNode` method of cluster.js